modules/service-account-auto-secret-removed.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * authentication/using-service-accounts-in-applications.adoc
+// * pods/nodes-pods-secrets.adoc
+
+:_content-type: CONCEPT
+[id="auto-generated-sa-token-secrets_{context}"]
+= About automatically generated service account token secrets
+
+When a service account is created, a service account token secret is automatically generated for it. This service account token secret, along with an automatically generated dockercfg secret, is used to authenticate to the internal {product-title} registry.
```

Comment:
We should mention that these secrets may not be generated in a future version and therefore noone should rely on their existence.